addons/base/views.py

```diff
@@ -339,12 +346,19 @@ def get_auth(auth, **kwargs):
                         if isinstance(node, Preprint):
                             metric_class = get_metric_class_for_action(action, from_mfr=from_mfr)
                             if metric_class:
+                                sloan_flag = {
+                                    'sloan_coi': request.cookies.get(SLOAN_COI),
+                                    'sloan_data': request.cookies.get(SLOAN_DATA),
+                                    'sloan_prereg': request.cookies.get(SLOAN_PREREG),
+                                    api_settings.SLOAN_ID: request.cookies.get(api_settings.SLOAN_ID)
```

Comment:
how about separate settings for the impact field name and the cookie name? those two concepts don't need to be entangled

or, for consistency, just hard-code the impact field name like the others
```suggestion
                                    'sloan_id': request.cookies.get(api_settings.SLOAN_ID_COOKIE_NAME)
```